#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_apache2_other.dpatch by  <debian@onerussian.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Tiny patch to mention apache2 possibility

@DPATCH@

diff -x '*~' -Naur fail2ban-0.6.1.pre/config/fail2ban.conf.iptables fail2ban-0.6.1.post/config/fail2ban.conf.iptables
--- fail2ban-0.6.1.pre/config/fail2ban.conf.iptables	2006-05-27 23:23:51.000000000 -0400
+++ fail2ban-0.6.1.post/config/fail2ban.conf.iptables	2006-05-27 23:27:30.000000000 -0400
@@ -270,6 +270,7 @@
 # Option:  logfile
 # Notes.:  logfile to monitor.
 # Values:  FILE  Default:  /var/log/apache/error.log
+# Other.: /var/log/apache2/error.log
 #
 logfile = /var/log/apache/error.log
 
